application/classes/Ushahidi/Core.php

```diff
@@ -542,6 +542,34 @@ public static function init()
 		$di->setter['Ushahidi_Listener_PostSetListener']['setRepo'] =
 			$di->lazyGet('repository.notification.queue');
 
+		// Login rate limiter
+		$di->set('ratelimiter.login', $di->lazyNew('BehEh\Flaps\Flap'));
+
+		$di->params['BehEh\Flaps\Flap'] = [
+			'storage' => $di->lazyGet('ratelimiter.storage'),
+			'name' => 'login'
+		];
+
+		// Rate limiter storage
+		$di->set('ratelimiter.storage', $di->lazyNew('BehEh\Flaps\Storage\DoctrineCacheAdapter'));
```

Comment:
- Skip DB support.. sounds like extra effort.
- yup memcache and filesystem as config options would be good.